lib/dotcom/trip_plan/itinerary_groups.ex

```diff
@@ -22,8 +22,8 @@ defmodule Dotcom.TripPlan.ItineraryGroups do
   @spec from_itineraries([Itinerary.t()], Keyword.t()) :: [ItineraryGroup.t()]
   def from_itineraries(itineraries, opts \\ []) do
     itineraries
-    |> Enum.group_by(&unique_legs_to_hash/1)
-    |> Enum.map(&drop_hash/1)
+    |> Enum.group_by(&{&1.accessible?, unique_legs_to_hash(&1)})
+    |> Enum.map(&elem(&1, 1))
```

Comment:
Oh weird - Github lost my other comment here:

**style:** This is purely aesthetic, but I find
```elixir
fn {_, itinerary_list} -> itinerary_list end
```

a lot easier to understand than
```elixir
&elem(&1, 1)
```